<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>sysCoach</class>
 <widget class="QDialog" name="sysCoach">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1291</width>
    <height>651</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1291</width>
    <height>651</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1291</width>
    <height>651</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>教练系统</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(232, 232, 232);</string>
  </property>
  <widget class="QWidget" name="left_widget" native="true">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>0</y>
     <width>171</width>
     <height>661</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:#DDDDDD</string>
   </property>
   <widget class="QListWidget" name="listFunc">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>131</width>
      <height>441</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QListWidget#listFunc{
    outline: 0px;
    border:0px;
    min-width: 120px;
    color: Black;
    background: #DDDDDD;
}

QListWidget#listFunc::Item{
     height:30px;
}
QListWidget#listFunc::Item:selected {
    background: rgb(49, 194, 124);
    border-radius:1.5px;
   
}
</string>
    </property>
    <item>
     <property name="text">
      <string>添加教练人员</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>resource/add_c.svg</normaloff>../resource/add_c.svg</iconset>
     </property>
    </item>
    <item>
     <property name="text">
      <string>更改教练信息</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>resource/up_c.svg</normaloff>../resource/up_c.svg</iconset>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 14pt &quot;黑体&quot;;</string>
    </property>
    <property name="text">
     <string>菜单栏</string>
    </property>
   </widget>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>50</y>
     <width>1061</width>
     <height>561</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 14pt &quot;黑体&quot;;</string>
     </property>
     <property name="text">
      <string>添加教练人员</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>641</width>
       <height>501</height>
      </rect>
     </property>
     <property name="title">
      <string>教练信息</string>
     </property>
     <widget class="QTextEdit" name="te_cmsg">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>140</y>
        <width>561</width>
        <height>311</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>110</y>
        <width>72</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>简单介绍：</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>41</y>
        <width>561</width>
        <height>54</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>姓名：</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1" colspan="2">
        <widget class="QLineEdit" name="le_c_name">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>联系方式：</string>
         </property>
        </widget>
       </item>
       <item row="0" column="4">
        <widget class="QLineEdit" name="le_c_phone">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);</string>
         </property>
        </widget>
       </item>
       <item row="0" column="5">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>性别：</string>
         </property>
        </widget>
       </item>
       <item row="0" column="6">
        <widget class="QComboBox" name="cb_c_gender">
         <item>
          <property name="text">
           <string>男</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>女</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>类别：</string>
         </property>
        </widget>
       </item>
       <item row="1" column="4">
        <widget class="QComboBox" name="cb_a_type">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);</string>
         </property>
         <item>
          <property name="text">
           <string>轮滑</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>平衡车</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>均可</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>职级：</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1" colspan="2">
        <widget class="QComboBox" name="cb_a_rank">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);</string>
         </property>
         <item>
          <property name="text">
           <string>助教</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>初级</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>中级</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>高级</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>680</x>
       <y>50</y>
       <width>351</width>
       <height>501</height>
      </rect>
     </property>
     <property name="title">
      <string>操作栏</string>
     </property>
     <widget class="QPushButton" name="btn_confirm">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>60</y>
        <width>221</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>确认录入</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_clear">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>160</y>
        <width>221</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>清空输入</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 14pt &quot;黑体&quot;;</string>
     </property>
     <property name="text">
      <string>更改教练信息</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="geometry">
      <rect>
       <x>680</x>
       <y>50</y>
       <width>351</width>
       <height>481</height>
      </rect>
     </property>
     <property name="toolTip">
      <string extracomment="发发"/>
     </property>
     <property name="title">
      <string>操作栏</string>
     </property>
     <widget class="QPushButton" name="btn_confirm_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>300</y>
        <width>201</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>确认修改</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_remove">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>390</y>
        <width>201</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>删除教练</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_22">
      <property name="geometry">
       <rect>
        <x>21</x>
        <y>71</y>
        <width>75</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>搜索状态：</string>
      </property>
     </widget>
     <widget class="QLabel" name="lb_result">
      <property name="geometry">
       <rect>
        <x>103</x>
        <y>71</y>
        <width>221</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QTableView" name="tv_search_coach">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>90</y>
        <width>311</width>
        <height>192</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>311</width>
        <height>30</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_3" columnstretch="0,0,0">
       <item row="0" column="2">
        <widget class="QPushButton" name="btn_search">
         <property name="text">
          <string>搜索</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="le_search_term">
         <property name="toolTip">
          <string/>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);</string>
         </property>
         <property name="placeholderText">
          <string>输入&quot;*&quot;查询全部</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>关键字：</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="btn_verify">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>300</y>
        <width>101</width>
        <height>151</height>
       </rect>
      </property>
      <property name="text">
       <string>认证</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>641</width>
       <height>481</height>
      </rect>
     </property>
     <property name="title">
      <string>教练信息</string>
     </property>
     <widget class="QTextEdit" name="te_cmsg_2">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>140</y>
        <width>561</width>
        <height>311</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>110</y>
        <width>72</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>简单介绍：</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>41</y>
        <width>561</width>
        <height>54</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <widget class="QLabel" name="label_16">
         <property name="text">
          <string>姓名：</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1" colspan="2">
        <widget class="QLineEdit" name="le_c_name_3">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QLabel" name="label_17">
         <property name="text">
          <string>联系方式：</string>
         </property>
        </widget>
       </item>
       <item row="0" column="4">
        <widget class="QLineEdit" name="le_c_phone_3">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);</string>
         </property>
        </widget>
       </item>
       <item row="0" column="5">
        <widget class="QLabel" name="label_18">
         <property name="text">
          <string>性别：</string>
         </property>
        </widget>
       </item>
       <item row="0" column="6">
        <widget class="QComboBox" name="cb_c_gender_3">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <item>
          <property name="text">
           <string>男</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>女</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QLabel" name="label_19">
         <property name="text">
          <string>类别：</string>
         </property>
        </widget>
       </item>
       <item row="1" column="4">
        <widget class="QComboBox" name="comboBox_3">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);</string>
         </property>
         <item>
          <property name="text">
           <string>轮滑</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>平衡车</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>均可</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_20">
         <property name="text">
          <string>职级：</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1" colspan="2">
        <widget class="QComboBox" name="comboBox_4">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);</string>
         </property>
         <item>
          <property name="text">
           <string>助教</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>初级</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>中级</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>高级</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QLabel" name="lb_status">
     <property name="geometry">
      <rect>
       <x>890</x>
       <y>540</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>当前状态：未认证</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="lb_head">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>10</y>
     <width>281</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 28pt &quot;楷体&quot;;</string>
   </property>
   <property name="text">
    <string>教 练 系 统</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
